flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/local/LocalExecutorITCase.java

```diff
@@ -476,6 +520,43 @@ public void testStreamQueryExecutionTable() throws Exception {
 		executeStreamQueryTable(replaceVars, query, expectedResults);
 	}
 
+	@Test(timeout = 90_000L)
+	public void testStreamQueryExecutionTableMultipleTimes() throws Exception {
+		final URL url = getClass().getClassLoader().getResource("test-data.csv");
+		Objects.requireNonNull(url);
+
+		final Map<String, String> replaceVars = new HashMap<>();
+		replaceVars.put("$VAR_PLANNER", planner);
+		replaceVars.put("$VAR_SOURCE_PATH1", url.getPath());
+		replaceVars.put("$VAR_EXECUTION_TYPE", "streaming");
+		replaceVars.put("$VAR_RESULT_MODE", "table");
+		replaceVars.put("$VAR_UPDATE_MODE", "update-mode: append");
+		replaceVars.put("$VAR_MAX_ROWS", "100");
+
+		final String query = "SELECT scalarUDF(IntegerField1), StringField1 FROM TableNumber1";
```

Comment:
Add query with "`" to verify the temporary table is dropped?

Reply:
Already have that, see `test-table`.

Reply:
the `test-table` case is only queried once, so didn't verify the temporary table "SELECT * FROM `test-table`" is dropped?

Reply:
I have add the tests that execute the query multiple times.